website/docs/cloud-docs/api-docs/private-registry/modules.mdx

```diff
@@ -256,7 +256,8 @@ Properties without a default value are required.
 | --------------------------------------------- | ------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `data.type`                                   | string  |         | Must be `"registry-modules"`.                                                                                                                                    |
 | `data.attributes.vcs-repo.identifier`         | string  |         | The repository from which to ingress the configuration.                                                                                                          |
-| `data.attributes.vcs-repo.oauth-token-id`     | string  |         | The VCS Connection (OAuth Connection + Token) to use as identified. You can obtain this ID from the [oauth-tokens](/terraform/cloud-docs/api-docs/oauth-tokens) endpoint. |
+| `data.attributes.vcs-repo.oauth-token-id`     | string  |         | The VCS Connection (OAuth Connection + Token) to use as identified. You can obtain this ID from the [oauth-tokens](/terraform/cloud-docs/api-docs/oauth-tokens) endpoint. This value must not be specified if `github-app-installation-id` is specified. |
```

Comment:
I'm not sure what `to use as identified` means here so I didn't make a specific suggestion, but I think that could use some clarifying. 

```suggestion
| `data.attributes.vcs-repo.oauth-token-id`     | string  |         | The VCS Connection (OAuth Connection + Token) to use as identified. Get this ID from the [oauth-tokens](/terraform/cloud-docs/api-docs/oauth-tokens) endpoint. You can not specify this value if `github-app-installation-id` is specified. |
```